src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/Snapshots/RestoreLogger.cs

```diff
@@ -2,18 +2,17 @@
 
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.VisualStudio.ProjectSystem.Logging;
 using NuGet.SolutionRestoreManager;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.PackageRestore
 {
     internal static class RestoreLogger
     {
-        public static void BeginNominateRestore(IProjectLogger logger, string fullPath, IVsProjectRestoreInfo2 projectRestoreInfo)
+        public static void BeginNominateRestore(IProjectDiagnosticOutputService logger, string fullPath, IVsProjectRestoreInfo2 projectRestoreInfo)
         {
             if (logger.IsEnabled)
             {
-                using IProjectLoggerBatch batch = logger.BeginBatch();
+                using var batch = new BatchLogger(logger);
```

Comment:
Style question: Should we prefer `using BatchLogger batch = new(logger);`?

In general we try to avoid using `var`.

Reply:
I'm happy either way. My feeling is that this kind of thing should be enforced by analyzers where possible to maintain consistency. We do have a lot of use of `var` in the code today, especially where the type is evident.